spec/controllers/books_spec.rb

```diff
@@ -0,0 +1,48 @@
+require 'rails_helper'
+
+describe BooksController, type: :controller do
+  let(:user) { create(:user) }
+
+  before { login(user) }
+
+  describe 'GET #index' do
+    context 'When fetching all the books' do
```

Comment:
We should also test that this endpoint is paginated. Check this link that explains how to test it using a helper https://github.com/Wolox/wor-paginate#test-helpers